adapters/mediafuse/usersync.go

```diff
@@ -0,0 +1,12 @@
+package mediafuse
+
+import (
+	"text/template"
+
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/usersync"
+)
+
+func NewMediafuseSyncer(temp *template.Template) usersync.Usersyncer {
+	return adapters.NewSyncer("mediafuse", 0, temp, adapters.SyncTypeRedirect)
```

Comment:
Would we want to use Adtelligent's GVL ID here? I'm not sure of the legal requirements for aliases.

Reply:
yeah